junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/EngineDiscoveryOrchestrator.java

```diff
@@ -45,7 +48,7 @@
  *
  * @since 1.7
  */
-@API(status = INTERNAL, since = "1.7", consumers = "testkit")
+@API(status = INTERNAL, since = "1.7", consumers = { "testkit", "suite" })
```

Comment:
The usage of `consumers` is rather inconsistent. What is the preferred usage?

```
@API(status = EXPERIMENTAL, since = "1.3", consumers = "org.junit.platform.engine.support.hierarchical")
@API(status = INTERNAL, since = "5.8", consumers = "org.junit.vintage.**")
@API(status = INTERNAL, since = "1.7", consumers = { "testkit", "suite" })
```